src/Slider/TimeSlider.jsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import {
+  isArray
+} from 'lodash';
+
+import {
+  Slider
+} from 'antd';
+
+/**
+ * Customized slider that uses ISO 8601 time strings as input.
+ *
+ * @class The TimeSlider
+ * @extends React.Component
+ */
+class TimeSlider extends React.Component {
+
+  static propTypes = {
+    /**
+     * A class name string to use on surrounding div.
+     * @type {String}
+     */
+    className: PropTypes.string,
+
+    /**
+     * Whether to allow range selection.
+     * @type {Boolean}
+     */
+    useRange: PropTypes.bool,
+
+    /**
+     * The default value(s).
+     * @type {any}
```

Comment:
Is there any other type than `Array<String>` or `String` supported?

Reply:
Nope, docs adapted.